drivers/gpio/Kconfig.xec

```diff
@@ -5,6 +5,12 @@
 
 config GPIO_XEC
 	bool "XEC Microchip GPIO driver"
-	depends on SOC_FAMILY_MEC
+	depends on SOC_SERIES_MEC1501X
 	help
 	  Enable the Microchip XEC gpio driver.
+
+config GPIO_XEC_V2
```

Comment:
@scottwcpg to be considered in upcoming drivers using GPIO_XEC_ECIA instead V2 to distinguish both GPIO approaches